### ISSUE TITLE
Second "ORDER BY" in SELECT with GROUP and INCLUDES (MSSQL)

### ISSUE THREAD

**Issue by Trapholov** [sequelize/sequelize#5932]
## What you are doing?

```js
Customer.hasMany(A);
A.belongsTo(Customer);
```

Changes in 3.23.2 caused me a problem.
In 3.23.0 i used this query:

```js
A.findAll({
    where: {
      customerId: {
        $ne: null
      }
    },
    attributes: [
      [sequelize.fn('MIN', sequelize.col('A.customerId')), 'customerId'],
      [sequelize.fn('SUM', sequelize.col('A.amount')), 'amountSum']
    ],
    group: 'customerId',
    limit: 5,
    hasIncludeRequired: true,
    include: [{
      model: Customer,
      attributes: [
        [sequelize.fn('MIN', sequelize.col('Customer.email')), 'email']
      ]
    }],
    order: [ sequelize.literal('amountSum DESC')],
    raw: true
}
```
Without hasIncludeRequired: true it was the same in 3.23.0 as now in 3.23.2. And this parameter is not working anymore. I know this is some sort of "hack" so.

## What do you expect to happen?
There should not be two ORDER BY clauses.

## What is actually happening?
Sequelize adds the second "ORDER BY" clause to query:

```sql
SELECT
MIN([A].[customerId]) AS [customerId],
SUM([A].[amount]) AS [amountSum],
MIN([Customer].[email]) AS [Customer.email]
FROM [A] AS [A]
LEFT OUTER JOIN [Customers] AS [Customer]
ON [A].[CustomerId] = [Customer].[Id]
WHERE [A].[OperationStatus] = 3
AND [A].[CustomerId] IS NOT NULL
GROUP BY customerId
ORDER BY amountSum DESC
ORDER BY [Id] <--
OFFSET 0 ROWS
FETCH NEXT 5 ROWS ONLY;
```

I think there should not be default order by Id if options.group exists.
Or what am i doing wrong?

__Dialect:__ mssql
__Database version:__ 12.0.2000.8 on Azure
__Sequelize version:__ 3.23.2

**Comment by AckerApple** [comment]
I'm going to try my dern' best right now to make my first pull request, to fix for this. It's an easy fix:

At Line:692 of lib/dialects/mssql/query-generator.js INSERT THIS CODE:
```
if(model.name){
  fragment += this.quoteIdentifier(model.name) + '.';
}
```

Wish me luck!

**Comment by AckerApple** [comment]
I made the pull request to fix

https://github.com/sequelize/sequelize/pull/6181

**Comment by sushantdhiman** [comment]
Fixed by #6181
